submarine-workbench/workbench-server/pom.xml

```diff
@@ -34,11 +34,16 @@
   <version>0.3.0-SNAPSHOT</version>
   <name>Submarine: Workbench Server</name>
 
+  <properties>
+    <commons-httpclient.version>3.1</commons-httpclient.version>
+  </properties>
+
   <dependencies>
+
     <dependency>
       <groupId>org.apache.submarine</groupId>
       <artifactId>commons-utils</artifactId>
-      <version>0.3.0-SNAPSHOT</version>
+      <version>${submarine.version}</version>
```

Comment:
This is good, Can you create a jira and Refactoring the `pom.xml` of the this project?

Reply:
I've already updated all dependencies, which are 0.3.0-SNAPSHOT , to ${submarine.version} in this PR.